source/guides/clear/mixer.rst

```diff
@@ -345,61 +345,100 @@ Refresh your web server to see the update content for mix version 20.
 You can also look in ~/mixer/update/www/<mix version> to see the update
 content in your workspace.
 
-
 Example 4: Build an image
 =========================
 
 This example shows how to build a bootable image containing the
 :command:`kernel-kvm`, :command:`os-core`, and the :command:`os-core-update`
-bundles from Example 2. Underneath, mixer uses `ister`_ to generate the
+bundles from Example 2. Underneath, mixer uses `clr-installer`_ to generate the
 image.
 
 #. Change directory into your mix.
 
 #. Configure image.
 
-   Edit the ister configuration file for your image to include all of the bundles you want pre-installed in the image. If this is the first time creating an image, first get a copy of the
-   :file:`release-image-config.json` template file:
+   Create a YAML configuration file to specify aspects of your image
```

Comment:
@bktan8 @mvincerx This PR looks good. But if we want to keep things simpler we could avoid the curl steps in Example 4. 
If the installer config yaml file is not found or provided, Mixer will automatically create one based on the bundles in the mix. So Example 4 could just be doing `sudo mixer build image` after the steps in Example 2.

Reply:
@reaganlo - I know what you mean and I considered it too.  But I wanted to give the reader a quick intro to creating a YAML and how to tweak their image instead of just dumping every bundle into an image.

Reply:
@reaganlo and I discussed his approach, equally valid. @bktan8, let's use your approach.  I understand why you want to make this process explicit--esp. for first-time users of `mixer`.